cachi2/core/extras/envfile.py

```diff
@@ -35,8 +35,8 @@ def based_on_suffix(cls, filepath: Path):
     @classmethod
     def _suffixes_repr(cls) -> str:
         return ", ".join(
-            f"{suffix}[=={fmt}]" if suffix != fmt else suffix
-            for suffix, fmt in cls.__members__.items()
```

Comment:
:open_mouth: good catch

They added a [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) in 3.11 (with the behavior of the current (str, Enum)) and changed the existing behavior.

Reply:
Would we prefer to use StrEnum here instead? :thinking:

Reply:
If we were 3.11 only, I'd say yes, but for now I think your change is a better solution

Reply:
Oh right, of course :facepalm: 
You can tell that I've already mentally moved on from 3.9/3.10 :slightly_smiling_face: